### DISCUSSION TITLE
Design: enhance the "download" button

### DISCUSSION THREAD

**Discussion by hugoroy** [tosdr/tosdr.org#102] [Ideas] [answered]
- make alternative download links and alternative browser name addon/extension depending on browser used; (for instance: add-on for Mozilla Firefox ; extension for Google Chrome, otherwise just use the current link to downloads.html
- add a  cropped/zoomed-screenshot according to the web browser, somewhere near the download button so that people understand what the extension does. I guess for this we could move the download button left.

Other improvements welcome!

**Comment by JustinBack** [comment, marked as answer]
Now In beta at https://github.com/PixelcatProductions/CrispCMS-ToS-DR/discussions/44
